lib/constants.dart

```diff
@@ -0,0 +1,6 @@
+//String constants for Kudos history
+String completedBeforeDeadline(int days_diff) => "completed $days_diff before deadline";
+String completedOnTime = "completed on time";
+String completedAfterDeadline(int days_diff) => "completed $days_diff after deadline";
+String completeTaskWithNoDeadline(String task) => "completed $task";
+scoreReducedForTask(String task) => "Score reduced for $task";
```

Comment:
Umm ideally you should have added EOF here as well but merging it for now, take this up in another PR

Reply:
sure sir, will create a new pr for this.